verilog/CST/macro_test.cc

```diff
@@ -174,5 +178,47 @@ TEST(GetFunctionFormalPortsGroupTest, WithFormalPorts) {
   }
 }
 
+TEST(FindAllMacroDefinitions, MacroName) {
+  constexpr int kTag = 1;  // value doesn't matter
+  const SyntaxTreeSearchTestCase kTestCases[] = {
+      {""},
+      {"`define ", {kTag, "PRINT_STRING"}, "(str1) $display(\"%s\\n\", str1)"},
```

Comment:
Suggested test cases:

macro definition appearing inside the following bodies:

* `module`
* `package`
* `class`
* `function`
* `task`

(Technically all macro definitions live inside the same global preprocessor namespace, no matter where they appear.)